src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```
@@ -215,6 +215,7 @@ OPF_096=Non-linear content must be reachable, but found no hyperlink to "%1$s".
 OPF_096b=No hyperlink was found to non-linear document "%1$s", please check that it can be reached from scripted content.
 OPF_097=Resource "%1$s" is listed in the manifest, but no reference to it was found in content documents.
 OPF_098=The "href" attribute must not reference resources via elements in the package document itself, but found URL "%1$s".
+OPF_099=The package document must not include itself in its manifest. 
```

A minor clarity suggestion:

```suggestion
OPF_099=The manifest must not list the package document . 
```